packages/webapp/components/FooterNavBar.tsx

```diff
@@ -65,15 +65,15 @@ export default function FooterNavBar({
     'shadow-[0_4px_30px_rgba(0,0,0.1)]',
   );
 
-  const Component = isNewMobileLayout ? FooterNavBarV1 : FooterNavBarControl;
+  const Component = isLaptop ? FooterNavBarControl : FooterNavBarV1;
 
   return (
     <div
       className={classNames(
         'fixed !bottom-0 left-0 z-2 w-full',
-        isNewMobileLayout
-          ? 'footer-navbar bg-gradient-to-t from-background-subtle from-70% to-transparent px-2 pt-2'
-          : post && 'bg-blur-bg backdrop-blur-20',
+        post && 'laptop:bg-blur-bg laptop:backdrop-blur-20',
+        !isLaptop &&
```

Comment:
This is going to be true all the time since as mentioned this file is not available for laptop. This means we don't need the condition anymore.

Reply:
This file is actually available on laptop, as with showNav we only use it to conditionally render only parts of this file, I'll move showNav to actually conditionally render this whole file so this works correctly

Reply:
Yeah, it seems that some parts were rendered conditionally only. Now that we completely do not render this footer, we should maybe take the Scroll to top component out so it would still be available on Tablet and Laptop.

Reply:
Brought back show nav on the FooterNavBarTabs, but fixed classes to show the scroll to top properly